app/javascript/packages/document-capture/components/button-to.jsx

```diff
@@ -33,7 +33,7 @@ function ButtonTo({ url, method, children, ...buttonProps }) {
       {createPortal(
         <form ref={formRef} method="post" action={url}>
           <input type="hidden" name="_method" value={method} />
-          <input type="hidden" name="authenticity_token" value={csrf} />
+          {csrf && <input type="hidden" name="authenticity_token" value={csrf} />}
```

Comment:
TODO: `csrf` is currently typed as `string`, but if we're assuming it can be `null` (see e40b020b2ec5256cf2103a4ff183424073f77d03), we should update the type to reflect this.

Reply:
> TODO: `csrf` is currently typed as `string`, but if we're assuming it can be `null` (see [e40b020](https://github.com/18F/identity-idp/commit/e40b020b2ec5256cf2103a4ff183424073f77d03)), we should update the type to reflect this.

See #5796